app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,11 @@
+class FoodsController < ApplicationController
+  def search
+    @total_results = FoodDetailsService.new.keyword_search(params[:q])[:totalHits]
+    foods = FoodDetailsService.new.keyword_search(params[:q])[:foods][0..9]
+    @food_details = foods.pluck(:gtinUpc, :description, :brand_owner, :ingredients)
```

Comment:
You're sending a hash to the view instead of a ruby object. Create a poro to store this information.